### PR TITLE
toggle help

### LINKED ISSUE #34: Add button to navbar to show/hide sample commands

### PR DESCRIPTION
**Pull request by JasperCheung**
Toggle help to nav bar, I couldn't keep the the x on the alert and still have it toggle. 
Issue: Closes #34 

### Frontend:
- tie toggle to nav bar

### CODE REVIEW

File: frontend/src/components/Navbar.jsx [hunk 1]
```diff
@@ -22,8 +31,10 @@ class SHRDLUNavbar extends Component {
             <Nav.Link href="/">Chat</Nav.Link>
             <Nav.Link href="/">Explore</Nav.Link>
             <Nav.Link href="/">Login</Nav.Link>
+            <Nav.Link onClick={this.handleClick}> Toggle Help</Nav.Link>
           </Nav>
         </Navbar>
+        <SampleCommands show={this.state.show} />
```
Comment (joshcwinton) [inline frontend/src/components/Navbar.jsx:37]: does this mean the help will show globally?

**Review by joshcwinton — APPROVED** [review]
Looks good, I think we might want to think about whether the help should appear on the other pages we plan to have 

💸

**Comment by joshcwinton** [comment]
merge at will

File: frontend/src/components/Navbar.jsx (hunk 1 above)
Comment (JasperCheung) [reply frontend/src/components/Navbar.jsx:37]: yeah, we can add a if statement to conditionally render it once we add more pages. Or just move it into a button on Chat.jsx, but we can do that later

**Pull request merged**